src/services/account/services/SupportJWTService.js

```diff
@@ -10,11 +10,10 @@ const { authenticationSecret, audience: audienceName } = require('../../authenti
 const accountModel = require('../model');
 const logger = require('../../../logger');
 
-const { getRedisClient, redisSetAsync } = require('../../../utils/redis');
+const { getRedisClient } = require('../../../utils/redis');
```

Comment:
please remove drect dependencies to the redisclient, can we instead here use mustCheckTokenIsWhitelisted() from whitelist.js? this maybe directly should happen in addTokenToWhitelist instead but not in this file.